### DISCUSSION TITLE
[iOS] How to get View and ShadowNode from tag

### DISCUSSION THREAD

**Discussion by itgyy** [reactwg/react-native-new-architecture#209] [Q&A]
**How do I dynamically add an existing native view to another native view?**
In the old framework I could get a View based on **viewForReactTag**, **shadowViewForReactTag** for nodes, dynamically add child nodes to the parent node, and dynamically add a view with addSubView, but with the new one I don't know how to do that.

**Comment by cortinico** [comment]
> How do I dynamically add an existing native view to another native view?

Can you clarify what you're attempting to do?

**Reply by itgyy** [reply]
I wanted to create a full-screen playback function similar to react-native video control, but maintain the state of video, so I thought of addSubview in ios, which can only change the view level and frame, without having to recreate and render a new view. I did this in the old framework by dynamically adding and modifying nodes and views, but the new framework doesn't know how to handle it
